### PR TITLE
Display names has been changed

### PR DESCRIPTION
**Pull request by recena**
@reviewbybees specially @hrmpw and @ndeloof 

Related with https://github.com/jenkinsci/cloudbees-folder-plugin/pull/38 and https://github.com/jenkinsci/github-branch-source-plugin/pull/32

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
   <groupId>com.cloudbees.jenkins.plugins</groupId>
   <artifactId>docker-custom-build-environment</artifactId>
-  <name>CloudBees Docker Custom Build Environment Plugin</name>
+  <name>Docker Custom Build Environment Plugin. Maintained by CloudBees, Inc.</name>
```
Comment (oleg-nenashev) [inline pom.xml]: I do not think it's required in the name. Description from index.jelly should be enough

Comment (ndeloof) [inline pom.xml]: Does CloudBees still plan to maintain it, or are we looking for a new maintainer ?

Comment (recena) [inline pom.xml]: @oleg-nenashev Please, take a look to the other PRs related. This was what we decided.

Comment (recena) [inline pom.xml]: @ndeloof I don't know about it. I had this task in my opensource backlog 😄

Comment (oleg-nenashev) [inline pom.xml]: @recena I think the decisions you refer to are outdated. Please follow-up with OSS Team  internally before going forward.

**Comment by stephenc** [comment]
:bug: by @oleg-nenashev AIUI

**Comment by oleg-nenashev** [comment]
Yep

**Comment by recena** [comment]
@oleg-nenashev Reason?

**Comment by recena** [comment]
@oleg-nenashev If this decision is obsolete or deprecated, I can close the PR /cc @hrmpw

File: pom.xml (hunk 1 above)
Comment (jglick) [inline pom.xml]: > I do not think it's required in the name. Description from index.jelly should be enough

Agreed.

Comment (recena) [inline pom.xml]: @jglick I did what we did in other plugins. If the criterial have changed... It is fine for me. I'll update the PR.

**Comment by recena** [comment]
@oleg-nenashev @stephenc ping?

**Comment by recena** [comment]
@oleg-nenashev @stephenc ping II?

**Comment by stephenc** [comment]
@hrmpw Can you provide guidance?

**Comment by recena** [comment]
@hrmpw ping?

**Comment by HRMPW** [comment]
Sorry, @recena  I didn't see any notifications for this PR until the last one.

**Pull request closed without merging**